source/manual/data-gov-uk-deployment.html.md

```diff
@@ -75,6 +75,14 @@ Builds can take up to 10 minutes. You can view progress in GitHub Actions under
 
 Deployments typically take 5 minutes. You can view progress in Argo CD.
 
+### How to check that your release has been successfully deployed
+
+In order to validate that your release has been deployed you will need to check the relevant pod in the relevant environment for Argo CD.
+
+For changes to `CKAN` you will need to look under the `ckan` application and for the `Find` app you will need to look under the `datagovuk` application.
+
+If the Argo app health status is not green then the deployment has not been successful and will need investigating before you continue testing the web application as you will not be testing your release but the previous release as that will still be running.
```

Comment:
I'd avoid using colours to be more inclusive. 

```suggestion
If the Argo app health status is not "Healthy" then the deployment has not been successful and will need to be investigated by checking the pod's logs. You should not continue testing the web application in this case, as the new release hasn't been successfully deployed, and you will be testing the previous release instead.
```

Reply:
There are probably a number of ways to investigate why a pod is failing so I'm going to expland on this